src/plugins/embeddable/public/plugin.tsx

```diff
@@ -177,6 +197,103 @@ export class EmbeddablePublicPlugin implements Plugin<EmbeddableSetup, Embeddabl
     }
   }
 
+  private telemetry = (state: EmbeddableInput) => {
```

Comment:
Is there a use case to have the telemetry API on the client? AFAICT it's only called on the server

Reply:
just for reasons of completion. if you feel it should be removed i can do so.

Reply:
I think we should remove it as long as we don't have a real use case - mostly for simplicity reasons.